stub.go

```diff
@@ -1 +1,5 @@
 package pets
+
+import (
+	"github.com/spf13/cobra"
```

Comment:
we probably want to vendor this and check it into git. Can you run `dep init` and `dep ensure` to download the dependency?